pkg/transfer/image/imagestore.go

```diff
@@ -325,6 +335,11 @@ func (is *Store) Get(ctx context.Context, store images.Store) (images.Image, err
 	return store.Get(ctx, is.imageName)
 }
 
+// ListExportImageNames returns a list of image names to export
+func (is *Store) ListExportImageNames() []string {
+	return is.exportImageNames
+}
```

Comment:
@dmcgowan I decided to save the names of images to be exported in the image store.